Buildings/Resources/Scripts/Dymola/Experimental/RadiantControl/SlabTempSignal/Validation/BaseClasses/Validation/ForHiChi.mos

```diff
@@ -0,0 +1,10 @@
+simulateModel("Buildings.Experimental.RadiantControl.SlabTempSignal.Validation.BaseClasses.Validation.ForHiChi", stopTime=31536000, method="cvode",  tolerance=1E-06, resultFile="ForHiChi");
+createPlot(id=1, position={55, 50, 1186, 817}, y={"forecastHighChicago.TForecastHigh"}
+ y={"forHiSF.TForecastHigh", "weaDat2.weaBus.TDryBul"}, 
+ range={0.0, 31536000, 250, 320},
+ autoscale = true,
+ autoerase = true,
+ autoreplot = true,
+ grid=true,
+ color=true,
+ filename="ForHiChi.mat")
```

Comment:
Per [our wiki](https://github.com/lbl-srg/modelica-buildings/wiki/Unit-Tests), a blank line is needed at the end of each mos script.